scripts/zkai_attention_kv_sixteen_head_fused_softmax_table_native_gate.py

```
@@ -0,0 +1,543 @@
+        raise AttentionKvSixteenHeadFusedSoftmaxTableGateError("fused envelope byte size drift")
+    digest = hashlib.blake2b(envelope_bytes, digest_size=32).hexdigest()
+    cache_key = (digest, len(envelope_bytes))
+    cached = _NATIVE_VERIFY_CACHE.get(cache_key)
+    expected = expected_fused_verifier_summary(len(envelope_bytes))
+    if cached is not None:
+        assert_native_verifier_summary(cached, expected, "cached native fused verifier summary")
+        return
+    with tempfile.TemporaryDirectory() as tmp:
+        envelope_path = pathlib.Path(tmp) / "fused-envelope.json"
+        envelope_path.write_bytes(envelope_bytes)
+        with subprocess.Popen(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)